includes/libraries/LDAP/adLDAP/adLDAP.php

```diff
@@ -88,15 +88,15 @@ class adLDAP
     */
     protected $_domain_controllers = array ("dc01.mydomain.local");
 
-    /**
-    * Optional account with higher privileges for searching
-    * This should be set to a domain admin account
+    /** Service account for searching
+    * Which should NOT be a domain admin
+    * Because that's FUCKING RETARDED
```

Comment:
I have never used a domain admin to query :+1: 
... but your comment makes me laugh :D :smiling_imp: